adapters/Tapjoy/README.md

```diff
@@ -0,0 +1,25 @@
+Steps to generate universal static library and framework for iOS Devices and
```

Comment:
Please move this README to BuildScript folder and update it to match our existing README style. Please follow the example README [here](https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/adapters/Unity/BuildScript/README.md)